api/handlers.go

```diff
@@ -531,17 +487,17 @@ func notFound(ctx echo.Context, err string) error {
 ///////////////////////
 
 // fetchAssets fetches all results and converts them into generated.Asset objects
-func (si *ServerImplementation) fetchAssets(ctx context.Context, options idb.AssetsQuery) ([]generated.Asset, error) {
-	assetchan := si.db.Assets(ctx, options)
+func (si *ServerImplementation) fetchAssets(ctx context.Context, options idb.AssetsQuery) ([]generated.Asset, uint64, error) {
```

Comment:
it would be nicer to have `round uint64` named in the return values (here and a few places below)

Reply:
I'm not a fan of named return values, but I added comments. Hope this is fine.